### PR TITLE
update instructions - rename library

### PR DESCRIPTION
**Pull request by 1devNdogs**
Separated changes, format and update docs and instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,25 +44,25 @@ Build the library.
 
 ```javascript
 // Import the Subspace.js library.
-import { SubspaceClient, Identity } from "subspace.js";
+import { SubspaceClient, Identity } from 'subspace.js'
```
Comment (nazar-pc) [inline README.md]: BTW, we need to rename this from `subspace.js` to `@subspace/subspace`

File: README.md [hunk 2]
```diff
@@ -44,25 +44,25 @@ Build the library.
...
 
 // Generate an Identity from node.js
-const identity = await Identity.fromUri(mnemonic);
+const identity = await Identity.fromUri(mnemonic)
```
Comment (nazar-pc) [inline README.md]: Why deleting semicolons everywhere?

File: README.md [hunk 3]
```diff
@@ -43,7 +43,7 @@ Build the library.
 ## Usage
 
 ```javascript
-// Import the Subspace.js library.
+// Import the library.
 import { SubspaceClient, Identity } from 'subspace.js'
```
Comment (nazar-pc) [inline README.md]: This line needs to be updated now too

File: examples/html/src/index.ts [hunk 4]
```diff
@@ -59,4 +59,4 @@ async function init() {
     (document.getElementById('injectedAccount') as HTMLHeadingElement).innerHTML = address
 }
 
-setTimeout(init, 2000)
\ No newline at end of file
+setTimeout(init, 2000)
```
Comment (nazar-pc) [inline examples/html/src/index.ts]: Conflict resolution brought this timeout back

File: examples/dapp/package.json [hunk 5]
```diff
@@ -17,7 +17,7 @@
       "not dead",
       "not op_mini all"
     ],
-    "development": [
+    "development": [
```
Comment (nazar-pc) [inline examples/dapp/package.json]: Extra whitespace at the end is not needed

File: README.md (hunk 2 above)
Comment (nazar-pc) [reply README.md]: I'm this was resolved without any fixes and comments, un-resolving then

Comment (1devNdogs) [reply README.md]: Sorry i will re create my responses, cancelled my own review losing all comments.

As response to this, i was just trying to make it simpler, like less characters for the eyes. think it looks better. I can rollback to the previous format.

File: README.md (hunk 3 above)
Comment (1devNdogs) [reply README.md]: It is updated in the last commit.

File: README.md (hunk 2 above)
Comment (nazar-pc) [reply README.md]: Not sure how it is easier to read :confused:
Personally I automatically look for the remaining of the code unless I see semicolon, so the benefit is questionable at best, I'd leave it as it was instead.

File: examples/dapp/package.json (hunk 5 above)
Comment (1devNdogs) [reply examples/dapp/package.json]: Updated.
https://github.com/subspace/subspace.js/pull/7/commits/a4c452705b9649a03814358769335e8cdffca89e

**Review by nazar-pc — DISMISSED** [review]

File: examples/html/src/index.ts (hunk 4 above)
Comment (1devNdogs) [reply examples/html/src/index.ts]: Oh, i think you where checking the **prev** commit, 

https://github.com/subspace/subspace.js/blob/445cc2b061aa3dc0cffddef76e8a49c08babb2a7/examples/html/src/index.ts#L62

**Review by nazar-pc — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (1devNdogs) [reply README.md]: Sure, i made the changes ;)

**Pull request merged**